flocker/apiclient/_client.py

```diff
@@ -0,0 +1,90 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Client for the Flocker REST API.
+"""
+
+from uuid import UUID
+
+from zope.interface import Interface
+
+from pyrsistent import PClass, field, pmap_field
+
+
+class Dataset(PClass):
```

Comment:
1. We should probably switch to `PClass`, yeah. It was added after we wrote that code.
2. I am writing this was expectation that it may someday be standalone, yes.